build.gradle.kts

```diff
@@ -24,7 +24,36 @@ dependencies {
     testImplementation(platform("org.junit:junit-bom:5.10.0"))
```

Comment:
the idea behind this task it is to separate unit tests and functional tests.
I see that you create "functionalTest" project directory. But you need to move `CaloriesControllerTest` to new module.
After this we will be able to separate dependencies.
So here we should remove `testImplementation("org.springframework.boot:spring-boot-starter-test")` and add `functionalTestImplementation("org.springframework.boot:spring-boot-starter-test")` instead.